README.md

```diff
@@ -11,25 +12,38 @@ ITKElastix
 Overview
 --------
 
-Provides an [ITK](https://www.itk.org) Python interface to [elastix](https://elastix.lumc.nl/), a toolbox for rigid and nonrigid registration of images.
+Provides an [ITK](https://www.itk.org) Python, JavaScript, and WebAssembly interface to [elastix](https://elastix.lumc.nl/), a toolbox for rigid and nonrigid registration of images.
 
 elastix is open source software, based on the well-known [Insight Toolkit (ITK)](https://discourse.itk.org). The software consists of a collection of algorithms that are commonly used to solve (medical) image registration problems. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application.
 
+[👨‍💻 **Live JavaScript API Demo** ✨](https://itk-wasm-elastix-app-js.on.fleek.co/ ':include :type=iframe width=100% height=800px')
+
 Installation
 ------------
 
-Install cross-platform binary Python packages with [pip](https://pypi.org/project/pip/):
+Install cross-platform native binary Python packages with [pip](https://pypi.org/project/pip/):
+
+```sh
+pip install itk-elastix
+```
+
+*Experimental* WebAssembly packages can be installed across platforms with:
```

Comment:
Suggest `WebAssembly packages` -> `WebAssembly Python packages`, clarifies that `pip install`ed packages are for Python in case users are unfamiliar with WebAssembly